packages/apps/plugins/plugin-settings/src/components/SettingsDialog.tsx

```diff
@@ -34,7 +34,7 @@ export const SettingsDialogContent = () => {
   // TODO(burdon): Common treatment for dialogs (e.g., shrink height to fit, mobile, etc.)
   return (
     <Dialog.Content classNames={['h-full md:max-is-[40rem] overflow-hidden']}>
-      <Dialog.Title>{t('settings dialog title', { ns: 'os' })}</Dialog.Title>
+      <Dialog.Title>{t('settings dialog title')}</Dialog.Title>
```

Comment:
Note, PluginList below is the kind of thing we should have as a higher level component (react-ui-list) so that we have keynav/selection/highlight.